cmd/dependabot/internal/cmd/update_test.go

```diff
@@ -1,13 +1,14 @@
 package cmd
 
 import (
-	"github.com/dependabot/cli/internal/model"
 	"net/http"
 	"os"
 	"reflect"
 	"strings"
 	"testing"
 	"time"
+
+	"github.com/dependabot/cli/internal/model"
```

Comment:
My editor makes this change automatically on save. I'm 👍 for it, but probably wouldn't notice/care if my editor didn't do it.